src/d3d8/d3d8_main.cpp

```diff
@@ -51,6 +51,10 @@ extern "C" {
     return S_OK;
   }
 
+  DLLEXPORT void __stdcall DebugSetMute() {
+    dxvk::Logger::debug("D3D8: DebugSetMute: Stub");

```

Comment:
Can we make sure this is only logged once, especially if the game is spamming it?

Reply:
Can do, but I forgot how that was being done :sweat_smile: .